### PR TITLE
Define how other members override the value imported via sdfRef

### PR DESCRIPTION
**Pull request by cabo**

### CODE REVIEW

File: sdf.md [hunk 1]
```diff
@@ -707,6 +708,25 @@ For example, this reference:
...
+Additional members may be present with the intention to override parts
+of the referenced map.
+More formally, the semantics of a JSON map given that contains an
+sdfRef member is defined to be as if the following steps were performed:
+
+1. The JSON map given that contains the sdfRef member is copied into a
+   variable named "patch".
+2. The sdfRef member of the copy in "patch" is removed.
+3. the JSON pointer that is the value of the sdfRef member is
+   dereferenced and copied into a variable named "original".
```
Comment (akeranen) [inline sdf.md:721]:
```suggestion
   dereferenced and the result is copied into a variable named "original".
```

**Review by akeranen — APPROVED** [review]
I think the mechanism is good. Here's a set of editorial suggestions that *may* clarify the text.

File: sdf.md [hunk 2]
```diff
@@ -707,6 +708,25 @@ For example, this reference:
 
 creates a new definition "temperatureProperty" that contains all of the qualities defined in the definition at /sdfData/temperatureData.
 
+The sdfRef member need not be the only member of a map.
+Additional members may be present with the intention to override parts
+of the referenced map.
+More formally, for an JSON map that contains an
```
Comment (akeranen) [inline sdf.md:714]:
```suggestion
More formally, for a JSON map that contains an
```

**Review by akeranen — APPROVED** [review]

**Comment by cabo** [comment]
After a few good edits prompted by Ari and off-list positive feedback, I merged this.

**Pull request merged**